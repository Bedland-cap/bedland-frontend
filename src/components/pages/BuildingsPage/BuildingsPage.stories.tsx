import type { Meta, StoryObj } from '@storybook/react';
import GlobalStyle from 'theme/globalStyles';
import ThemeProvider from 'theme/ThemeContext';
import BuildingsPage from './BuildingsPage';

const meta = {
  title: 'pages/BuildingsPage',
  component: BuildingsPage,
  tags: ['autodocs'],
  parameters: {
    backgrounds: { default: 'light' },
  },
  decorators: [
    (Story) => (
      <ThemeProvider>
        <GlobalStyle />
        {Story()}
      </ThemeProvider>
    ),
  ],
} satisfies Meta<typeof BuildingsPage>;

export default meta;

type Story = StoryObj<typeof meta>;

export const Default: Story = {};
