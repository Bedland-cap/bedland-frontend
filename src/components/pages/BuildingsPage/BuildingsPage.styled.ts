import styled from 'styled-components';

export const SearchInputContainer = styled.div`
  display: flex;
  justify-content: flex-end;
  margin-top: 0.5rem;
`;

export const BuildingsPageContent = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
`;
