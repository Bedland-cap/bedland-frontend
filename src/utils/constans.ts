export const HEADER_HEIGHT = 70;
export const FOOTER_HEIGHT = 52;
