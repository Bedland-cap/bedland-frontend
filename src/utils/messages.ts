const ERROR_MESSAGES = {
  resetPasswordPage: {
    notWithPasswordsRules:
      'Password is incorrect, please try again after reviewing the rules above',

    passwordsDoesNotMatchUp: 'Passwords do not match to each other',
  },
};

export default ERROR_MESSAGES;
