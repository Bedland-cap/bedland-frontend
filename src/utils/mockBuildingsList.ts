import { Building } from 'store/reducers/building/building_slice';

const mockBuildingsList: Building[] = [
  {
    id: '1',
    name: 'Berry',
    address: 'Berry Street 21',
    managerId: '1',
    floors: '5',
  },
  {
    id: '2',
    name: 'Miedziana',
    address: 'Miedziana Street 3',
    managerId: '1',
    floors: '7',
  },
  {
    id: '3',
    name: 'Seaport',
    address: 'Seaport Avenue 51',
    managerId: '2',
    floors: '5',
  },
  {
    id: '4',
    name: 'Desert',
    address: 'Desert Street 2X',
    managerId: '1',
    floors: '6',
  },
  {
    id: '5',
    name: 'Lakewood',
    address: 'Lakewood Avenue 57',
    managerId: '3',
    floors: '10',
  },
];

export default mockBuildingsList;
