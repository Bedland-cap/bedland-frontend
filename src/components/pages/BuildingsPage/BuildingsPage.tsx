import SearchInput from 'components/atoms/SearchInput/SearchInput';
import BuildingsList from 'components/organisms/BuildingsList/BuildingsList';
import Page from 'components/templates/Page/Page';
import SortButton from 'components/molecules/SortButton/SortButton';
import * as Styled from './BuildingsPage.styled';

const sortOptions = ['name', 'city', 'date added'];

const BuildingsPage = () => (
  <Page title='Buildings'>
    <Styled.BuildingsPageContent>
      <Styled.SortSearchContainer>
        <SortButton text='Sort Buildings' sortOptions={sortOptions} />
        <SearchInput placeholder='Search buildings...' />
      </Styled.SortSearchContainer>
      <BuildingsList />
    </Styled.BuildingsPageContent>
  </Page>
);

export default BuildingsPage;
