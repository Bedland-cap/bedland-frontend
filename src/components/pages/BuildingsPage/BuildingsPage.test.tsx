import { renderWithProviders } from 'utils/test.utils';
import BuildingsPage from './BuildingsPage';

describe('BuildingsPage', () => {
  it('renders correctly', () => {
    const page = renderWithProviders(<BuildingsPage />, {});
    expect(page).toMatchSnapshot();
  });
});
