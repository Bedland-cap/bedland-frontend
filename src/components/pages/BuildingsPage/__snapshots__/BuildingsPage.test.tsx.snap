// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BuildingsPage renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": .c5 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #fff;
  border-radius: 8px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 2.5rem;
  padding: 0.5rem;
  width: 17.5rem;
}

.c5:focus-within {
  outline: 1px solid #4553DD;
}

.c7 {
  background-color: transparent;
  color: #4B4B4B;
  height: 1.75rem;
  width: 15rem;
}

.c7::-webkit-input-placeholder {
  color: #767676;
}

.c7::-moz-placeholder {
  color: #767676;
}

.c7:-ms-input-placeholder {
  color: #767676;
}

.c7::placeholder {
  color: #767676;
}

.c6 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: 0.583rem;
  margin-right: 0.771rem;
}

.c9 {
  background-color: #fff;
  border-radius: 8px;
  box-shadow: 0px 5px 8px #4B4D6220;
  color: #4b4b4b;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 20rem;
}

.c10 {
  height: 11rem;
  object-fit: cover;
  padding: 0.5rem;
  width: 100%;
}

.c11 {
  padding: 0.5rem 1.031rem 0.938rem;
}

.c12 {
  margin-top: 1rem;
}

.c13 {
  background-color: #4553DD;
  border-radius: 8px;
  color: #fff;
  font-size: 1rem;
  font-weight: 600;
  line-height: 140%;
  padding: 0.563rem;
  width: 100%;
}

.c13:hover {
  background-color: #909AFF;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
  gap: 2rem;
  margin-top: 1.5rem;
}

.c0 {
  -webkit-align-items: start;
  -webkit-box-align: start;
  -ms-flex-align: start;
  align-items: start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 1rem 1.5rem;
  width: 100%;
}

.c1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 1rem;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  width: 100%;
}

.c2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  width: 100%;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  margin-top: 0.5rem;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<body>
    <div>
      <div
        class="c0"
      >
        <div
          class="c1"
        >
          <h2
            style="font-family: Poppins, sans-serif; font-weight: 700; font-size: 1.5rem; line-height: 1.4; color: rgb(0, 0, 0);"
          >
            Buildings
          </h2>
        </div>
        <div
          class="c2"
        >
          <div
            class="c3"
          >
            <div
              class="c4"
            >
              <div
                class="c5"
              >
                <div
                  class="c6"
                >
                  <svg
                    fill="none"
                    height="16"
                    viewBox="0 0 16 16"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <mask
                      id="mask-search"
                    />
                    <path
                      d="M14.6666 14.6666L11.2916 11.2916M11.2916 11.2916C11.8333 10.7499 12.263 10.1068 12.5562 9.39907C12.8494 8.6913 13.0003 7.93272 13.0003 7.16664C13.0003 6.40055 12.8494 5.64197 12.5562 4.9342C12.263 4.22644 11.8333 3.58334 11.2916 3.04164C10.7499 2.49994 10.1068 2.07023 9.39907 1.77707C8.6913 1.4839 7.93272 1.33301 7.16664 1.33301C6.40055 1.33301 5.64197 1.4839 4.9342 1.77707C4.22644 2.07023 3.58334 2.49994 3.04164 3.04164C1.94762 4.13566 1.33301 5.61946 1.33301 7.16664C1.33301 8.71381 1.94762 10.1976 3.04164 11.2916C4.13566 12.3857 5.61946 13.0003 7.16664 13.0003C8.71381 13.0003 10.1976 12.3857 11.2916 11.2916Z"
                      stroke="#767676"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="1.15972"
                    />
                  </svg>
                </div>
                <input
                  class="c7"
                  placeholder="Search buildings..."
                />
              </div>
            </div>
            <div
              class="c8"
            >
              <div
                class="c9"
              >
                <img
                  alt="Building"
                  class="c10"
                  src=""
                />
                <div
                  class="c11"
                >
                  <h3
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                  >
                    Building #
                    1
                  </h3>
                  <p
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                  >
                    Berry Street 21
                  </p>
                  <div
                    class="c12"
                  >
                    <button
                      class="c13"
                    >
                      Open details
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="c9"
              >
                <img
                  alt="Building"
                  class="c10"
                  src=""
                />
                <div
                  class="c11"
                >
                  <h3
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                  >
                    Building #
                    2
                  </h3>
                  <p
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                  >
                    Miedziana Street 3
                  </p>
                  <div
                    class="c12"
                  >
                    <button
                      class="c13"
                    >
                      Open details
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="c9"
              >
                <img
                  alt="Building"
                  class="c10"
                  src=""
                />
                <div
                  class="c11"
                >
                  <h3
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                  >
                    Building #
                    3
                  </h3>
                  <p
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                  >
                    Seaport Avenue 51
                  </p>
                  <div
                    class="c12"
                  >
                    <button
                      class="c13"
                    >
                      Open details
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="c9"
              >
                <img
                  alt="Building"
                  class="c10"
                  src=""
                />
                <div
                  class="c11"
                >
                  <h3
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                  >
                    Building #
                    4
                  </h3>
                  <p
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                  >
                    Desert Street 2X
                  </p>
                  <div
                    class="c12"
                  >
                    <button
                      class="c13"
                    >
                      Open details
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="c9"
              >
                <img
                  alt="Building"
                  class="c10"
                  src=""
                />
                <div
                  class="c11"
                >
                  <h3
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                  >
                    Building #
                    5
                  </h3>
                  <p
                    style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                  >
                    Lakewood Avenue 57
                  </p>
                  <div
                    class="c12"
                  >
                    <button
                      class="c13"
                    >
                      Open details
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        data-testid="location-display"
      >
        /
      </div>
    </div>
  </body>,
  "container": .c5 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #fff;
  border-radius: 8px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 2.5rem;
  padding: 0.5rem;
  width: 17.5rem;
}

.c5:focus-within {
  outline: 1px solid #4553DD;
}

.c7 {
  background-color: transparent;
  color: #4B4B4B;
  height: 1.75rem;
  width: 15rem;
}

.c7::-webkit-input-placeholder {
  color: #767676;
}

.c7::-moz-placeholder {
  color: #767676;
}

.c7:-ms-input-placeholder {
  color: #767676;
}

.c7::placeholder {
  color: #767676;
}

.c6 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-left: 0.583rem;
  margin-right: 0.771rem;
}

.c9 {
  background-color: #fff;
  border-radius: 8px;
  box-shadow: 0px 5px 8px #4B4D6220;
  color: #4b4b4b;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 20rem;
}

.c10 {
  height: 11rem;
  object-fit: cover;
  padding: 0.5rem;
  width: 100%;
}

.c11 {
  padding: 0.5rem 1.031rem 0.938rem;
}

.c12 {
  margin-top: 1rem;
}

.c13 {
  background-color: #4553DD;
  border-radius: 8px;
  color: #fff;
  font-size: 1rem;
  font-weight: 600;
  line-height: 140%;
  padding: 0.563rem;
  width: 100%;
}

.c13:hover {
  background-color: #909AFF;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
  gap: 2rem;
  margin-top: 1.5rem;
}

.c0 {
  -webkit-align-items: start;
  -webkit-box-align: start;
  -ms-flex-align: start;
  align-items: start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 1rem 1.5rem;
  width: 100%;
}

.c1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 1rem;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  width: 100%;
}

.c2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  width: 100%;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  margin-top: 0.5rem;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div>
    <div
      class="c0"
    >
      <div
        class="c1"
      >
        <h2
          style="font-family: Poppins, sans-serif; font-weight: 700; font-size: 1.5rem; line-height: 1.4; color: rgb(0, 0, 0);"
        >
          Buildings
        </h2>
      </div>
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <div
            class="c4"
          >
            <div
              class="c5"
            >
              <div
                class="c6"
              >
                <svg
                  fill="none"
                  height="16"
                  viewBox="0 0 16 16"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <mask
                    id="mask-search"
                  />
                  <path
                    d="M14.6666 14.6666L11.2916 11.2916M11.2916 11.2916C11.8333 10.7499 12.263 10.1068 12.5562 9.39907C12.8494 8.6913 13.0003 7.93272 13.0003 7.16664C13.0003 6.40055 12.8494 5.64197 12.5562 4.9342C12.263 4.22644 11.8333 3.58334 11.2916 3.04164C10.7499 2.49994 10.1068 2.07023 9.39907 1.77707C8.6913 1.4839 7.93272 1.33301 7.16664 1.33301C6.40055 1.33301 5.64197 1.4839 4.9342 1.77707C4.22644 2.07023 3.58334 2.49994 3.04164 3.04164C1.94762 4.13566 1.33301 5.61946 1.33301 7.16664C1.33301 8.71381 1.94762 10.1976 3.04164 11.2916C4.13566 12.3857 5.61946 13.0003 7.16664 13.0003C8.71381 13.0003 10.1976 12.3857 11.2916 11.2916Z"
                    stroke="#767676"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="1.15972"
                  />
                </svg>
              </div>
              <input
                class="c7"
                placeholder="Search buildings..."
              />
            </div>
          </div>
          <div
            class="c8"
          >
            <div
              class="c9"
            >
              <img
                alt="Building"
                class="c10"
                src=""
              />
              <div
                class="c11"
              >
                <h3
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                >
                  Building #
                  1
                </h3>
                <p
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                >
                  Berry Street 21
                </p>
                <div
                  class="c12"
                >
                  <button
                    class="c13"
                  >
                    Open details
                  </button>
                </div>
              </div>
            </div>
            <div
              class="c9"
            >
              <img
                alt="Building"
                class="c10"
                src=""
              />
              <div
                class="c11"
              >
                <h3
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                >
                  Building #
                  2
                </h3>
                <p
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                >
                  Miedziana Street 3
                </p>
                <div
                  class="c12"
                >
                  <button
                    class="c13"
                  >
                    Open details
                  </button>
                </div>
              </div>
            </div>
            <div
              class="c9"
            >
              <img
                alt="Building"
                class="c10"
                src=""
              />
              <div
                class="c11"
              >
                <h3
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                >
                  Building #
                  3
                </h3>
                <p
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                >
                  Seaport Avenue 51
                </p>
                <div
                  class="c12"
                >
                  <button
                    class="c13"
                  >
                    Open details
                  </button>
                </div>
              </div>
            </div>
            <div
              class="c9"
            >
              <img
                alt="Building"
                class="c10"
                src=""
              />
              <div
                class="c11"
              >
                <h3
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                >
                  Building #
                  4
                </h3>
                <p
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                >
                  Desert Street 2X
                </p>
                <div
                  class="c12"
                >
                  <button
                    class="c13"
                  >
                    Open details
                  </button>
                </div>
              </div>
            </div>
            <div
              class="c9"
            >
              <img
                alt="Building"
                class="c10"
                src=""
              />
              <div
                class="c11"
              >
                <h3
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 700; font-size: 1.25rem; line-height: 1.75; letter-spacing: 1px; color: rgb(75, 75, 75);"
                >
                  Building #
                  5
                </h3>
                <p
                  style="font-family: Source Sans Pro, sans-serif; font-weight: 400; font-size: 1rem; line-height: 1.6; letter-spacing: 0; color: rgb(75, 75, 75);"
                >
                  Lakewood Avenue 57
                </p>
                <div
                  class="c12"
                >
                  <button
                    class="c13"
                  >
                    Open details
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      data-testid="location-display"
    >
      /
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;
